src/schema/user/types.js

```diff
@@ -39,6 +39,10 @@ const UserType = new GraphQLObjectType({
       type: GraphQLString,
       description: `A URL pointing to the logged user's public website.`,
     },
+    avatarUrl: {
+      type: GraphQLString,
+      description: 'User profile picture (avatar) URL.',
```

Comment:
```suggestion
      description: `The logged user's profile picture (avatar) URL.`,
```

Just to follow the description standard.